test/shared/fixtures.ts

```diff
@@ -584,6 +622,8 @@ interface EthVaultFixture {
   osToken: OsToken
   osTokenVaultController: OsTokenVaultController
   osTokenConfig: OsTokenConfig
+  ethValidatorsChecker: EthValidatorsChecker
+  gnoValidatorsChecker: GnoValidatorsChecker
```

Comment:
No need to add it to the fixture here. Lets deploy the contract only in the test